assets/js/components/ExecutionResults/CheckResultDetail/GatheredFacts.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+import ListView from '@components/ListView';
+import ObjectTree from '@components/ObjectTree/ObjectTree';
+
+function GatheredFacts({ gatheredFacts = [] }) {
+  const facts = gatheredFacts.map(({ name, value, type, message }) => {
+    const isFactError = !!type;
+
+    return {
+      title: name,
+      content: value,
+      render: (factValue) =>
+        isFactError ? (
+          <span className="text-red-500">{message}</span>
+        ) : (
+          <ObjectTree className="mt-3" data={factValue || {}} />
+        ),
```

Comment:
```suggestion
      render: (factValue = {}) =>
        isFactError ? (
          <span className="text-red-500">{message}</span>
        ) : (
          <ObjectTree className="mt-3" data={factValue} />
        ),
```

I'm a bit fried at the moment, but would this work?